django/utils/version.py

```diff
@@ -13,10 +13,6 @@
 # or later". So that third-party apps can use these values, each constant
 # should remain as long as the oldest supported Django version supports that
 # Python version.
-PY36 = sys.version_info >= (3, 6)
-PY37 = sys.version_info >= (3, 7)
-PY38 = sys.version_info >= (3, 8)
-PY39 = sys.version_info >= (3, 9)
```

Comment:
Please check the comment that it's written above these constants. Django 4.2 support Python 3.8 and 3.9 so we shouldn't remove `PY38` and `PY39`.

Reply:
Thank you